CharacterCreation/03_Feats.txt

```diff
@@ -520,7 +526,7 @@ to the beginning of your next turn. Cannot be used in concert with Run 'N' Gun.
 
 Cover Slide [15 Nanites]:
 Prerequisites: DEX 4
-Move up to your speed and end next to a barrier behind which you can take 
+Move up to your speed and end next to a barrier behind which you can take

```

Comment:
Please put the space back, I was using it.

Reply:
kappa

Reply:
kappa? as in just K? I can't tell if you are serious or not.

Reply:
I'm totally joking haha. "kappa" is an expression coined by Twitch chat, usually used for comedic effect

Reply:
lol! Thanks for the explination. I waz teh confuz